### PR TITLE
Contract tests

### LINKED ISSUE #12: Contract tests
This project relies heavily on the google api - we should be aware if this api changes and breaks our app.
May be worth adding a contract test to the concourse pipeline so that we are notified if it changes functionality.

### PR DESCRIPTION
**Pull request by alice-carr**
This PR:
-  creates two contract tests for the google api 
- adds these contract tests to the concourse pipeline to run every 24 hours
- updates the documentation

closes #12

### CODE REVIEW

File: lambda/contract_test.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    """
+    Tests that the group admin returns a non empty dictionary
+    """
+    group_id = (
+        admin_google_client.groups()
+        .list(pageToken=None, domain="digital.cabinet-office.gov.uk", maxResults=1)
+        .execute()["groups"][0]["email"]
+    )
+    groups = groups_google_client.groups().get(groupUniqueId=group_id).execute()
+    assert groups != {}
```
Comment (0atman) [inline lambda/contract_test.py]:
```python
>>> bool({})
False
```
an empty dictionary is falsy, so you should test it with `assert groups` :ok_hand:

**Review by 0atman — CHANGES_REQUESTED** [review]

**Review by akinnane — APPROVED** [review]
Great

File: lambda/contract_test.py (hunk 1 above)
Comment (alice-carr) [reply lambda/contract_test.py]: cool - just changed that to `assert groups` ! :+1:

**Review by 0atman — APPROVED** [review]
Big fan :ok_hand:

**Pull request merged**